tests/test_config.cfg

```diff
@@ -1,4 +1,4 @@
 [default]
-model_names = invoice,receipt
+model_ids = las:model:014f1a205b4443e9ad1ea524ac963a3e, las:model:44deb3ea25aa443b852b0c8f96bcb4ee
```

Comment:
What are these values?

Reply:
just test values... This is an old design, maybe time to replace it with fixtures?